apps/.eslintCustomMessagesFormatter.js

```diff
@@ -47,5 +47,6 @@ module.exports = (results = []) => {
   });
 
   // then pass modified results array to the existing formatter
-  return defaultFormatter(results);
+  const defaultFormatter = await eslint.loadFormatter();
+  return defaultFormatter.format(results);
```

Comment:
These changes were required since ESLint 8 has breaking changes to its plugin API (deprecations in 7, now removed)